recipes/julia-mode

```diff
@@ -1,4 +1,4 @@
 (julia-mode
- :repo "JuliaLang/julia"
+ :repo "JuliaLang/julia-emacs"
  :fetcher github
- :files ("contrib/julia-mode.el"))
+ :files ("julia-mode.el"))
```

Comment:
I don't actually use emacs :grimacing: so I can't test...

(wanted to reply to an on -users with "MELPA package available", but noticed the url was outdated)